docs/index.rst

```diff
@@ -1,17 +1,67 @@
 .. include:: ../README.rst
    :end-before: About
 
+
 .. include:: ../README.rst
-   :start-after: -----
+   :start-line: 25
    :end-before: Features
 
-.. toctree::
-   :numbered:
 
-   introduction
-   cli
-   restapi
-   contributing
-   changes
-   license
-   authors
+.. include:: ../README.rst
+   :start-line: 32
```

Comment:
We could put a starting one-line sentence:

```
Features:

- ...
```

so that the itemised list would be introduced by something.